### PR TITLE
feat(web.hosting): move metrics to mimir

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-14513
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Move Webhosting metrics to mimir

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js [hunk 1]
```diff
@@ -150,7 +68,144 @@ export default class HostingTabAutomatedEmailsCtrl {
...
+                    hour: 'LT',
+                  },
+                },
+              },
+            },
+          },
+        });
+      },
+    );
   }
```
Comment (tibs245) [inline packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:209]: Possibility export this long request on a service ?

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js [hunk 2]
```diff
@@ -150,7 +68,144 @@ export default class HostingTabAutomatedEmailsCtrl {
...
+                    'hosting_tab_STATISTICS_none',
+                  ),
+                },
+                noDataMsg: {
+                  message: this.$translate.instant(
+                    'hosting_tab_STATISTICS_none',
+                  ),
+                },
+                findInBorderColorMap: () => '#848CBC',
+                findInBackgroundColorMap: () => '#EAECF4',
```
Comment (tibs245) [inline packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:100]: Maybe export on a constant ?

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js [hunk 3]
```diff
@@ -150,7 +68,144 @@ export default class HostingTabAutomatedEmailsCtrl {
...
+
+                  return fetch(url, { headers })
+                    .then((response) => {
+                      if (response.ok) {
+                        return response.json();
+                      }
+
+                      return null;
+                    })
+                    .then((response) => response.data);
```
Comment (tibs245) [inline packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:123]: Will should be `$http`

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js (hunk 1 above)
Comment (tibs245) [reply packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:209]: I see with another dev. You can leave here :)

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js (hunk 3 above)
Comment (lizardK) [reply packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:123]: No needs and avoid angularjs dependencies

**Review by rjamet-ovh — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js (hunk 3 above)
Comment (tibs245) [reply packages/manager/apps/web/client/app/hosting/automated-emails/hosting-automated-emails.controller.js:123]: Yes I understand your argument but it's the practice for this app

**Review by tibs245 — APPROVED** [review]

**Review by MaximeBajeux — APPROVED** [review]

**Pull request merged**